node/src/container_chain_spawner.rs

```diff
@@ -89,14 +89,16 @@ pub struct ContainerChainSpawnerState {
 
 pub struct ContainerChainState {
     /// Async callback that enables collation on this container chain
+    // We don't use it since we are always restarting container chains
+    #[allow(unused)]
```

Comment:
Interesting should we remove it already then?

Reply:
We could, but it will make it a bit harder to revert this change in the future

Reply:
no worries then